machine.go

```diff
@@ -1067,3 +1067,97 @@ func (m *Machine) CreateSnapshot(ctx context.Context, memFilePath, snapshotPath
 	m.logger.Debug("snapshot created successfully")
 	return nil
 }
+
+// CreateBalloon creates a balloon device if one does not exist
+func (m *Machine) CreateBalloon(ctx context.Context, amountMb int64, deflateOnOom bool, opts ...PutBalloonOpt) error {
+	balloon := models.Balloon{
+		AmountMb:     &amountMb,
+		DeflateOnOom: &deflateOnOom,
+	}
+	respNoContent, err := m.client.PutBalloon(ctx, &balloon, opts...)
+
+	if err == nil {
+		m.logger.Printf("Created balloon device with %v MB successful: %s", amountMb, respNoContent.Error())
```

Comment:
Isn't the Error() always `""`?

Reply:
I haven't really got an Error, but I think you are right. If the createBalloon request is successfully, there isn't anything. I just removed it.